e2e-tests/tests/e2e_flows.py

```diff
@@ -0,0 +1,310 @@
+import pytest
+import json
+
+from common.assertions import equal_json_strings
+from common.methods import analyze, anonymize, analyzer_supported_entities
+
+
+@pytest.mark.api
+def test_given_a_correct_analyze_input_then_anonymize_success():
```

Comment:
not critical: maybe: test_given_text_with_PII_then_analyze_and_anonymize_successfully
So the name will reflect this is a progress test